packages/cli/src/ResponseHelper.ts

```diff
@@ -154,8 +155,12 @@ export function send<T, R extends Request, S extends Response>(
 
 			sendSuccessResponse(res, data, raw);
 		} catch (error) {
-			if (error instanceof Error && isUniqueConstraintError(error)) {
-				error.message = 'There is already an entry with this name';
+			if (error instanceof Error) {
+				ErrorReporterProxy.getInstance().error(error);
+
+				if (isUniqueConstraintError(error)) {
+					error.message = 'There is already an entry with this name';
+				}
```

Comment:
In cases where we supplement the error, wouldn't it be useful for this info to be included in the error report?

Reply:
Might be a good idea, but in this case I'd prefer that the original error message be reported. Maybe a good place to do error chaining.